### PR TITLE
Bruno Branca Topic0 6 of html and 1st of css

### PR DESCRIPTION
**Pull request by brunobranca**

### CODE REVIEW

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <title>My page</title>
+  </head>
+  <body  style="background-color:#4f5151">
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html]: Avoid inline style. Could you move it to `style.css`?

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+<div class="header">
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html]: Fix the indentation

File: 00-HTML-CSS-basics/css/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
+html,body {
+
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/style.css:6]: This break line is not needed

File: 00-HTML-CSS-basics/css/style.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+   height: 100%;
+   width: 100%;
+}
+
+
+.header { text-align:center;
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/style.css]: move text-align rule to the next line

**Review by afiorenza — CHANGES_REQUESTED** [review]
Fix the indentation of the files and please alpha order the css rules. Thanks!

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link href="style.css" rel="stylesheet" type="text/css" >
+    <title>My page</title>
+  </head>
+  <body  >
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html]: You don't need the space between `body` and `>`

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+<div class="header">
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html]: You don't need to wrap the `h1` in a `div`. Just move the class to the `h1`.

Comment (afiorenza) [reply 00-HTML-CSS-basics/css/ex 2.1.html]: Check other places of this same file

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="sfooter">
+      <h1>Section Footer</h1>
+    </div>
+
+</div>
+
+<div class="aside">
+  <h1>aside</h1>
+</div>
+
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html:39]: Just one break line is okey

File: 00-HTML-CSS-basics/css/ex 2.1.html [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+
+<div class="footer">
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/ex 2.1.html]: Check indentation

File: 00-HTML-CSS-basics/css/style.css [hunk 9]
```diff
@@ -0,0 +1,104 @@
+html,body {
+   height: 100%;
+   width: 100%;
+   background-color:#4f5151
+}
+
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/style.css:6]: One break line is okey

File: 00-HTML-CSS-basics/css/style.css [hunk 10]
```diff
@@ -0,0 +1,104 @@
+html,body {
+   height: 100%;
```
Comment (afiorenza) [inline 00-HTML-CSS-basics/css/style.css]: Alpha order the rules

**Review by afiorenza — CHANGES_REQUESTED** [review]
Good job! just a few comments

File: 00-HTML-CSS-basics/css/ex 2.1.html (hunk 6 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/ex 2.1.html]: Checked

File: 00-HTML-CSS-basics/css/ex 2.1.html (hunk 1 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/ex 2.1.html]: done

File: 00-HTML-CSS-basics/css/style.css (hunk 3 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/style.css:6]: ok

File: 00-HTML-CSS-basics/css/style.css (hunk 4 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/style.css]: done

File: 00-HTML-CSS-basics/css/ex 2.1.html (hunk 5 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/ex 2.1.html]: done

File: 00-HTML-CSS-basics/css/ex 2.1.html (hunk 7 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/ex 2.1.html:39]: done

File: 00-HTML-CSS-basics/css/ex 2.1.html (hunk 8 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/ex 2.1.html]: done

File: 00-HTML-CSS-basics/css/style.css (hunk 10 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/style.css]: done

File: 00-HTML-CSS-basics/css/style.css (hunk 9 above)
Comment (brunobranca) [reply 00-HTML-CSS-basics/css/style.css:6]: ok

**Review by afiorenza — APPROVED** [review]
Good Job! Ready for merge

**Pull request merged**